app/src/NativeBase/Containers/ListContainer.tsx

```diff
@@ -36,25 +43,16 @@ import {
 } from '@/Services/modules/projects'
 import { EventsState, setEvents } from '@/Store/Events'
 import { ProjectsState, setProjects } from '@/Store/Projects'
-import Theme from '@/Theme/OldTheme'
+import underDevelopmentAlert from '@/Utils/UnderDevelopmentAlert'
 
 const ClearSearchLabel = styled.Text`
```

Comment:
Please can you remove these `styled` components entirely and replace them with NativeBase components e.g. Text where they're used below

Reply:
Ignore this too — I’m going to replace how search results work right after you’ve done this ticket